tests/test_js_graph.py

```diff
@@ -398,7 +399,7 @@ def test_get_geometry_plain(graph_cases):
     assert graph.get_geometry(graph.get_edge(*graph.get_vertices())) == \
         [40, 25, 50, 1]
     assert graph.get_geometry(graph.get_decorations()[0]) == [55, 20, 10, 10]
-    table_w, table_h = fix_table_size(102, 72)
+    table_w, table_h = fix_table_size(100, 72)
```

Comment:
I created #58 to find a way to better handle  `fix_table_size`.
=/ It is really anoying having to tune they areguments from time to time. Is it not?

Reply:
I don't know why this changed, doesn't seem related to my changes.
I don't really cared about it

Reply:
Yes... this is kind of flaky.